### ISSUE TITLE
Can't find the integration panel

### ISSUE THREAD

**Issue by johndevor** [AdsCryptoz22/EasyBuildSystem#5]
Trying to integrate with Opsive TPC. Reading the instructions, it says:

```
- Go to Main Editor in the menu items (Tools/Easy Build System/Main Editor) or CTRL+SHIFT+E.
- Go to integration panel and enable the Opsive Third Person Controller by clicking on the check box and wait the end compilation.
```
See screenshot below, but I can't find the integrations panel. Any idea? 

<img width="727" alt="Screen Shot 2020-02-25 at 6 38 35 PM" src="https://user-images.githubusercontent.com/75827/75297555-30304180-57fe-11ea-8399-ec177afa28c4.png">

**Comment by johndevor** [comment]
Using EBS v 4.6.7

**Comment by AdsCryptoz22** [comment]
Hi,

You can directly import (Opsive TPS Integration) which is in the package.
There is no longer any need to add SDS (https://docs.unity3d.com/Manual/PlatformDependentCompilation.html) for this integration.
However, if you use other integration this could cause you problem, some people have already reported this problem with the window on mac, I will try to fix it for the next release.
If you want to use other integrations do not hesitate to let me know, I will send you a simple guide to install them manually, directly via SDS.

Regards,

De : John
Envoyé le :26 February 2020 00:40
À : AdsCryptoz22/EasyBuildSystem
Cc : Subscribed
Objet :[AdsCryptoz22/EasyBuildSystem] Can't find the integration panel (#5)

Trying to integrate with Opsive TPC. Reading the instructions, it says:
- Go to Main Editor in the menu items (Tools/Easy Build System/Main Editor) or CTRL+SHIFT+E.
- Go to integration panel and enable the Opsive Third Person Controller by clicking on the check box and wait the end compilation.
See screenshot below, but I can't find the integrations panel. Any idea?

**Comment by johndevor** [comment]
Ah, thanks. If you have the instructions for uSurvival integration that would be appreciated!

**Comment by AdsCryptoz22** [comment]
Here how add scripting define symbols manually if the integration window is not shown.

You can go in your Project Settings > Player and go to field (Scripting Define Symbols).
Then go to the last line and add the specific symbols for integrations.

![Example](https://user-images.githubusercontent.com/40346381/75392924-5254bd00-58ed-11ea-9b91-b6b35fe6aafe.PNG)

Here the list of integration (do not forget to add ";" to split the integration).

Photon Network = EXITGAMESV1
Photon Network (V2) = EXITGAMESV2
Inventory Pro = INVENTORYPRODEVDOG
Rucksack = RUCKSACKDEVDOG
uSurvival = USURVIVALVIS2K
uMMORPG = UMMORPGVIS2K
uRPG = URPGVIS2K
Voxeland = VOXELAND

**Comment by AdsCryptoz22** [comment]
Fixed in the release (4.7.1).
